### DISCUSSION TITLE
ARM binaries missing from v0.43.0 onwards

### DISCUSSION THREAD

**Discussion by YOU54F** [aquasecurity/trivy#4972] [Bugs]
### Description

Release notes don't state arm binaries have been dropped

install.sh was updated as part of https://github.com/aquasecurity/trivy/pull/3985

as binaries have been published for arm up to date

### Desired Behavior

binaries are available or release notes to state dropping of platform

### Actual Behavior

404

### Reproduction Steps

```bash
Run audit workflow on https://github.com/pact-foundation/pact-broker-docker

https://github.com/pact-foundation/pact-broker-docker/actions/runs/5803686616/job/15732153239

or run a armv7 platform docker image and try and run the install.sh for this repo
```

### Target

Container Image

### Scanner

Vulnerability

### Output Format

None

### Mode

Standalone

### Debug Output

```bash
aquasecurity/trivy info checking GitHub for latest tag
aquasecurity/trivy info found version: 0.44.0 for v0.44.0/Linux/ARM
wget: server returned error: HTTP/1.1 404 Not Found
```

### Operating System

armv7 linux

### Version

```bash
cant download latest 0.44.0
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by YOU54F** [comment]
PR to remove from our pipeline

https://github.com/pact-foundation/pact-broker-docker/pull/142

note we only started providing native arm images ourselves very recently, so I am open to potentially dropping audit support for arm as its not officially supported by our project (its a YMMV affair)

**Reply by knqyf263** [reply]
https://github.com/aquasecurity/trivy/discussions/4789

**Reply by YOU54F** [reply]
Thanks @knqyf263 I did see this issue https://github.com/aquasecurity/trivy/issues/4952 and related pull request , but it doesn't state anything about arm, the commit doesn't either.

Appreciate the struggles on disk space.

Will close this down ta!

**Reply by knqyf263** [reply]
We addressed the disk space issue. It is time to revert arm binaries. Thanks for your patience.
https://github.com/aquasecurity/trivy/pull/4977
